keyboards/keebio/iris/keymaps/merrittlj/keymap.c

```diff
@@ -0,0 +1,156 @@
+// Copyright 2023 Lucas Merritt (merrittlj)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+#define _DVORAK 0
+#define _SHIFT 1
+#define _SYMBOL 2
+#define _FUNCTION 3
+#define _ADDITIONAL 4
+
+#define KC_LBRACE S(KC_LBRC)
+#define KC_RBRACE S(KC_RBRC)
+#define KC_LESSTHAN S(KC_COMMA)
+#define KC_MORETHAN S(KC_DOT)
+// #define KC_UNDERSCORE S(KC_MINUS)
+#define KC_PLUS S(KC_EQUAL)
+// #define KC_PIPE S(KC_BSLS)
+#define KC_DQUOTE S(KC_QUOTE)
+// #define KC_COLON S(KC_SCLN)
+#define KC_QMARK S(KC_SLSH)
+// #define KC_TILDE S(KC_GRAVE)
+#define KC_BANG S(KC_1)
+#define KC_AT S(KC_2)
+#define KC_HASH S(KC_3)
+// #define KC_DOLLAR S(KC_4)
+#define KC_PCNT S(KC_5)
+#define KC_AHEAD S(KC_6)
+#define KC_AND S(KC_7)
+// #define KC_ASTERISK S(KC_8)
+#define KC_LPAREN S(KC_9)
+#define KC_RPAREN S(KC_0)
```

Comment:
optimisation opportunity by utilising [shifted keycodes](https://docs.qmk.fm/#/keycodes_us_ansi_shifted).

Reply:
I didn't see the shifted keycodes on the all-keycodes page, let me fix all of them.